Automattic/wp-calypso
# Signup: remove locale suggestions flux implementation

This PR removes the flux implementation of `LocaleSuggestions`, and integrates the data-layer work completed in #21145.

## Testing
1. Make sure your browser sends multiple `Accept-Language` header values (in Chrome, for example, add a language at `chrome://settings/languages`)
2. In a new incognito window open `/start`
3. Click on an alternative locale

**Expectations**
A call to `https://public-api.wordpress.com/rest/v1.1/locale-guess` is made.

Alternative locale suggestions are displayed, and clicking on one changes the locale

![jan-28-2018 15-29-46](https://user-images.githubusercontent.com/6458278/35479022-6298d42a-0440-11e8-83fa-29f940c74423.gif)
